<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oxy Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons8-bookmark-page-512.png</normaloff>icons8-bookmark-page-512.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leProjectPath">
       <property name="placeholderText">
        <string>Проект для которого генерируется документация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSelectProjectDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="leProjectName">
     <property name="placeholderText">
      <string>Имя проекта в документации</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="leWorkingDir">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Рабочая директория (Директория с Solution)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSelectSolutionDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="leOutputDirectory">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Директория с документацией (пути будут относительно нее)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSelectDocDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="tbExclude">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Список исключений</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="tbHeaders">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Output после билда с ShowIncludes в студии</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Обработать вывод</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="teOutput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbGenerateFileDescription">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сгенерировать описание файлов</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbDoxyTemplate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Показать шаблон DoxyGen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbShowWarnings">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Запустить обработку doxygen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
